foreman.te

```diff
@@ -593,6 +601,8 @@ optional_policy(`
     tunable_policy(`passenger_run_foreman', `
         files_manage_generic_tmp_files(passenger_t)
         files_manage_generic_tmp_dirs(passenger_t)
+        logging_search_logs(passenger_t)
```

Comment:
Even in tunable block it should be in a nested optional block.

Reply:
It is. See line 600. I've learned that I can include tunable into optional but not vice versa, thus I moved it here instead.

Reply:
Sorry, not enough context on my screen, exactly 1 line less.